tests/assistant_spec.js

```diff
@@ -1148,145 +1142,177 @@ describe('TranslationAssistant (graphie)', function() {
     // NOTE(danielhollas): This test is flaky, not sure why
     it('should handle multiple graphies on multiple lines', function() {
         const allItems = [{
-            englishStr: `simplify ${graphie1}, answer ${graphie2}\n\n` +
-                `hints: ${graphie3}`,
-            translatedStr: `simplifyz ${graphie1}, answerz ${graphie2}\n\n` +
-                `hintz: ${graphie3}`,
+            englishStr: `simplify ${graphies[0]}, answer ${graphies[1]}\n\n` +
+                `hints: ${graphies[2]}`,
+            translatedStr: `simplifyz ${graphies[0]}, ans ${graphies[1]}\n\n` +
+                `hintz: ${graphies[2]}`,
         }];
         const itemsToTranslate = [{
-            englishStr: `simplify ${graphie4}, answer ${graphie5}\n\n` +
-                `hints: ${graphie6}`,
+            englishStr: `simplify ${graphies[3]}, answer ${graphies[4]}\n\n` +
+                `hints: ${graphies[5]}`,
             translatedStr: '',
         }];
 
         assertSuggestions(allItems, itemsToTranslate, [
-            `simplifyz ${graphie4}, answerz ${graphie5}\n\n` +
-            `hintz: ${graphie6}`,
+            `simplifyz ${graphies[3]}, ans ${graphies[4]}\n\n` +
+            `hintz: ${graphies[5]}`,
         ]);
     });
 
     it('should handle translations that re-order graphies', function() {
         const allItems = [{
-            englishStr: `simplify ${graphie1}, answer ${graphie2}`,
-            translatedStr: `answerz ${graphie2}, simplifyz ${graphie1}`,
+            englishStr: `simplify ${graphies[0]}, answer ${graphies[1]}`,
+            translatedStr: `answerz ${graphies[1]}, simplifyz ${graphies[0]}`,
         }];
         const itemsToTranslate = [{
-            englishStr: `simplify ${graphie3}, answer ${graphie4}`,
+            englishStr: `simplify ${graphies[2]}, answer ${graphies[3]}`,
             translatedStr: '',
         }];
 
         assertSuggestions(allItems, itemsToTranslate, [
-            `answerz ${graphie4}, simplifyz ${graphie3}`,
+            `answerz ${graphies[3]}, simplifyz ${graphies[2]}`,
         ]);
     });
 
     it('should handle strings that are only graphies', function() {
         const allItems = [{
-            englishStr: graphie1,
-            translatedStr: graphie1,
+            englishStr: graphies[0],
+            translatedStr: graphies[0],
         }];
         const itemsToTranslate = [{
-            englishStr: graphie2,
+            englishStr: graphies[1],
             translatedStr: '',
         }];
 
-        assertSuggestions(allItems, itemsToTranslate, [graphie2]);
+        assertSuggestions(allItems, itemsToTranslate, [graphies[1]]);
     });
 });
 
 describe('TranslationAssistant (image and graphie links)', function() {
+    // NOTE(danielhollas): Testing whether some test flakiness
+    // is caused by non-unique graphie or image links
+    it('tests should have unique Graphies', function() {
+        for (let i = 0; i < 6; i++) {
```

Comment:
Can you make `6` a file `const` please or use `graphies.length`?

Reply:
Yep, done.

Reply:
I still see `for (let i = 0; i < 6; i++)`?

Reply:
I'm sorry, I fixed it on one place but not here. Should be okay now.